### ISSUE TITLE
Add openSUSE Leap 15.0 and SLES 15.0 to the RID graph

### ISSUE THREAD

**Issue by joshfree** [dotnet/corefx#31026]
https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props#L79-L85 
```
    <!-- all opensuse releases are independent -->
    <RuntimeGroup Include="opensuse">
      <Parent>linux</Parent>
      <Architectures>x64</Architectures>
      <Versions>13.2;42.1;42.2;42.3</Versions>
      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
    </RuntimeGroup>
```

`Versions` should be extended to include 15.0.  See https://en.wikipedia.org/wiki/OpenSUSE_version_history 

https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props#L112-L116 
```
    <RuntimeGroup Include="sles">
      <Parent>linux</Parent>
      <Architectures>x64</Architectures>
      <Versions>12;12.1;12.2;12.3</Versions>
    </RuntimeGroup>
```
`Versions` should be extended here to include 15.0 as well.  See https://en.wikipedia.org/wiki/SUSE_Linux_Enterprise (release date: 16 July 2018)

**Comment by joshfree** [comment]
The instructions for how to add this are in the README at https://github.com/dotnet/corefx/tree/master/pkg/Microsoft.NETCore.Platforms

update ```runtimeGroups.props``` then run ```build.cmd -- /p:UpdateRuntimeFiles=true```

**Comment by AraHaan** [comment]
This issue looks like it needs closed. The change from looks of it was applied to branch master already.

**Comment by MattGal** [comment]
Yup I just didn't know this issue existed when Josh asked me to do that on the other PR.
